automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -98,7 +98,10 @@ def product_upgrade(product):
     If product is longrun then upgrade satellite, capsule and client
 
     :param string product: product name wanted to upgrade.
-
+    :param string sat_host: sat_host is satellite hostname.
+    :param string cap_hosts: cap_hosts is the capsule hosts.
+    :param string clients6: rhel 6 clients to be upgraded.
```

Comment:
Add a docstring saying, 'Optional in case of docker clients. If not given, the clients will be generated on docker'.